automation/util/setup.go

```diff
@@ -66,7 +65,7 @@ func LoadEnv(ctx context.Context) (err error) {
 	// Loading env variables
 	err = godotenv.Load("../../.env")
 	if err != nil {
-		return fmt.Errorf("Error: loadEnv() ended! %s", err)
+		return fmt.Errorf("error: loadEnv() ended! %s", err)
```

Comment:
I remember we talked about this before. Is making error messages lowercase a operator convention, ndb convention, go convention or coding common convention? Curious.

Reply:
I was getting go-static lint error/warnings, so it was suggested by my IDE.

Reply:
gotcha.